lib/rest/Twilio.js

```diff
@@ -246,11 +249,21 @@ Twilio.prototype.request = function request(opts) {
   var password = opts.password || this.password;
 
   var headers = opts.headers || {};
+
+  var pkgVersion = moduleInfo.version;
+  var osName = os.platform();
+  var osArch = os.arch();
+  var nodeVersion = process.version;
   headers['User-Agent'] = util.format(
-    'twilio-node/%s (node.js %s)',
-    moduleInfo.version,
-    process.version
+    'twilio-node/%s (%s %s) node/%s',
```

Comment:
Should this new standardization be applied across all libraries? (e.g. [twilio-python](https://github.com/twilio/twilio-python/blob/c25b21301c4f692c1e9d76774a906d67be51d175/twilio/rest/__init__.py#L115))

Reply:
https://twilio.slack.com/archives/CGQPL0RPH/p1625670427257100?thread_ts=1625623548.248000&cid=CGQPL0RPH